addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/blob/s3/S3BlobStoreConfiguration.java

```diff
@@ -256,6 +265,7 @@ protected AWSCredentialsProvider getAWSCredentialsProvider() {
     }
 
     protected ClientConfiguration getClientConfiguration() {
+        boolean proxyDisabled = Boolean.parseBoolean(Framework.getProperty(DISABLE_PROXY_PROPERTY));
```

Comment:
In case it remains a Framework property, _Framework#isBooleanPropertyTrue_ should do the work.